src/main/webapp/i18n/de/metis.json

```
@@ -7,10 +7,9 @@
             "addNewPost": "Neuer Beitrag",
             "answerNow": "Jetzt beantworten",
             "startDiscussion": "Diskussion beginnen",
-            "showAnswers": "Zeige {{ number }}",
+            "showMultipleAnswers": "Zeige {{ number }} Antworten",
+            "showSingleAnswer": "Zeige die Antwort",
```

```suggestion
            "showSingleAnswer": "Zeige 1 Antwort",
```
I'd suggest keeping the style of the one above for two reasons:
1. To stay consistent
2. It's not clear what "die" refers to. "Die" is quite direct. It should rather be "eine" and in that case, we can stick to the format

---

```suggestion
            "showSingleAnswer": "Zeige Antwort",
```
Alternatively, we can follow along with the English translation.
It doesn't bother me what but either choose one or the other (and change the other translation respectively to stay consistent.

---

Changed it to "Zeige Antwort", that's better, you're right